pxtlib/toolbox.ts

```diff
@@ -54,6 +63,7 @@ namespace pxt.toolbox {
         }
     }
 
+    let namespaceStyleBuffer: string = '';
```

Comment:
still used?

Reply:
Nope getting rid of it, and support to inject CSS by blocklyloader